src/assembler.jl

```diff
@@ -243,8 +292,8 @@ function _missing_sparsity_pattern_error(Krow::Int, Kcol::Int)
         "$(Kcol)] is missing in the sparsity pattern. Make sure you have called `K = " *
         "create_sparsity_pattern(dh)` or `K = create_sparsity_pattern(dh, ch)` if you " *
         "have affine constraints. This error might also happen if you are using " *
-        "`::AssemblerSparsityPattern` in a threaded assembly loop (you need to create an " *
-        "`assembler::AssemblerSparsityPattern` for each task)."
+        "`::CSCAssembler` in a threaded assembly loop (you need to create an " *
+        "`assembler::CSCAssembler` for each task)."
```

Comment:
```suggestion
        "the assembler in a threaded assembly loop (you need to create one " *
        "`assembler` for each task)."
```

or pass the type into this function, but seems overkill to me.